src/devices/Ssd13xx/tests/Commands/SetMultiplexRatioTests.cs

```diff
@@ -30,7 +30,6 @@ public void Get_Bytes(byte multiplexRatio, byte[] expectedBytes)
 
         [Theory]
         [InlineData(0x0E)]
-        [InlineData(0x40)]
```

Comment:
?

Reply:
As part of these changes I also ensured that all bindings that had test projects would run all tests correctly (we currently don't run them as part of CI). I did this because for some bindings (like both of the bindings I changed which were gpio extenders) this was a major change and I wanted to make sure that all tests would pass after the refactoring. That said, while running all bindings tests I found out that this one was failing, and I believe it has failed ever since it was added as I believe that this is a a value which wouldn't be considered out of range by the binding. That is why I removed this test case, in order to have all tests passing given that this one has been failing since day 1

Reply:
ok